app/__init__.py

```diff
@@ -230,6 +230,14 @@ def create_app(application):
     )
     logging.init_app(application)
 
+    import hashlib
+
+    for key in ("SECRET_KEY", "DANGEROUS_SALT"):
+        application.logger.info(
+            f"{key} Length: {len(application.config[key])}; "
+            f"SHA-1: {hashlib.sha1(application.config[key].encode('utf8'), usedforsecurity=False).hexdigest()}"
```

Comment:
Would the length alone not be enough of an indicator?

Is the use of the `SHA1` hash here just to get a sense that _some_ data is in place and to possibly compare it to hashing it ourselves separately and comparing that the two hashes match?  If that's the case, we should make sure we're following [NIST guidance](https://csrc.nist.gov/pubs/fips/202/final) here and using a method that's not vulnerable to any known attacks.

Reply:
I can adjust it accordingly. These hashes are not being used in any cryptographic way, but I can easily make it use something more in-line with the NIST guidance.

Reply:
The length could be enough. I just wanted to see that yes, there is a value there.

Reply:
I also made the logging a little more intuitive.